R/code.R

```diff
@@ -21,6 +22,10 @@ vt_use_test_code <- function(name, username = vt_username(), open = interactive(
                     ))
   }
 
+  # Add file to validation configuration
+  vt_add_file_to_config(filename = name, pkg = pkg, after = add_after,
```

Comment:
`pkg` is no longer an argument in `vt_add_file_to_config()`, which is probably why the test is failing CI.